openvpn-install.sh

```diff
@@ -110,6 +117,8 @@ if [[ -e /etc/openvpn/server.conf ]]; then
 		case $option in
 			1)
 			echo ""
+			echo "Here are the files that already exist,do not repeat that"
```

Comment:
What do these two lines do? How do they help in determining the OS version better?